responses/nl/HassGetState.yaml

```
@@ -1,4 +1,52 @@
 language: nl
 responses:
   intents:
-    HassGetState: {}
+    HassGetState:
+      one: "{{ slots.name | capitalize }} is {{ state.state_with_unit }}"
+      one_yesno: |
+        {% set translations =
+          {
+            "on": "aan",
+            "off": "uit",
+            "open": "open",
+            "closed": "dicht",
+            "opening": "aan het openen",
+            "closing": "aan het sluiten"
+          }
+        %}
+        {% if state.state_with_unit in translations.keys() %}
+          {% set translated = translations[state.state_with_unit] %}
+        {% endif %}
+        {% if query.matched: %}
+          Ja
+        {% else: %}
+          Nee, {{ slots.name }} is {{ translated if translated is defined else state.state_with_unit }}
+        {% endif %}
+      any: |
+        {% if query.matched: %}
+        Ja, {{ query.matched | map(attribute="name") | sort | join(", ") }}
+        {% else: %}
+        Nee
+        {% endif %}
+
+      all: |
+        {% if not query.unmatched: %}
+        Ja
+        {% else: %}
+        Nee, niet {{ query.unmatched | map(attribute="name") | sort | join(", ") }}
+        {% endif %}
+
+      which: |
+        {% if not query.matched: %}
+          Geen
+        {% else: %}
+          {% set match = query.matched | map(attribute="name") | sort | list %}
+          {% if match | length > 3 %}
```

Om te voorkomen dat we "en nog 1" zeggen, wat dan weer langer is dan de naam

```suggestion
          {% if match | length > 4 %}
```